jest.config.js

```diff
@@ -14,11 +14,12 @@
 
 module.exports = {
   preset: 'amex-jest-preset',
+  setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
   collectCoverageFrom: [
     'packages/**/*.{js,jsx}',
     '!**/node_modules/**',
     '!**/build/**',
-    '!packages/*/lib/**',
+    'packages/*/lib/**',
```

Comment:
this will now run coverage check on all packages lib files which could cause issues

Reply:
agreed, i'll update this to `src` since we are compiling this

Reply:
The library doesn't compile anything though.. it uses `esm` to load ES module syntax directly with an enhanced `require`, without any build steps. Doesn't `src` imply that there needs to be a step for building? As of right now, this is the only package using `lib` (which we can safely include as is in the config), and I understand the uniformity that `src` gives, it just seems mis labelled for a what the directory describes by name.

Some discussions on `src` vs `lib` I found:
https://gist.github.com/tracker1/59f2c13044315f88bee9
https://stackoverflow.com/questions/38426078/why-use-lib-vs-src-directory-names-in-javascript-which-is-standard